libraries/chain/include/eosio/chain/transaction.hpp

```diff
@@ -159,6 +159,8 @@ namespace eosio { namespace chain {
          local_pack_context_free_data();
       }
 
+      packed_transaction(const bytes& packed_txn, const vector<signature_type>& sigs, const bytes& packed_cfd, compression_type _compression);
```

Comment:
Don't add this just for a test. Use the constructor below. 
`packed_transaction( bytes&& packed_txn, vector<signature_type>&& sigs, bytes&& packed_cfd, compression_type _compression );`